src/API/Reports/Orders/Stats/Controller.php

```diff
@@ -39,30 +39,32 @@ class Controller extends \Automattic\WooCommerce\Admin\API\Reports\Controller {
-		$args                      = array();
-		$args['before']            = $request['before'];
-		$args['after']             = $request['after'];
-		$args['interval']          = $request['interval'];
-		$args['page']              = $request['page'];
-		$args['per_page']          = $request['per_page'];
-		$args['orderby']           = $request['orderby'];
-		$args['order']             = $request['order'];
-		$args['fields']            = $request['fields'];
-		$args['match']             = $request['match'];
-		$args['status_is']         = (array) $request['status_is'];
-		$args['status_is_not']     = (array) $request['status_is_not'];
-		$args['product_includes']  = (array) $request['product_includes'];
-		$args['product_excludes']  = (array) $request['product_excludes'];
-		$args['coupon_includes']   = (array) $request['coupon_includes'];
-		$args['coupon_excludes']   = (array) $request['coupon_excludes'];
-		$args['tax_rate_includes'] = (array) $request['tax_rate_includes'];
-		$args['tax_rate_excludes'] = (array) $request['tax_rate_excludes'];
-		$args['customer']          = $request['customer'];
-		$args['refunds']           = $request['refunds'];
-		$args['attribute_is']      = (array) $request['attribute_is'];
-		$args['attribute_is_not']  = (array) $request['attribute_is_not'];
-		$args['categories']        = (array) $request['categories'];
-		$args['segmentby']         = $request['segmentby'];
+		$args                       = array();
+		$args['before']             = $request['before'];
+		$args['after']              = $request['after'];
+		$args['interval']           = $request['interval'];
+		$args['page']               = $request['page'];
+		$args['per_page']           = $request['per_page'];
+		$args['orderby']            = $request['orderby'];
+		$args['order']              = $request['order'];
+		$args['fields']             = $request['fields'];
+		$args['match']              = $request['match'];
+		$args['status_is']          = (array) $request['status_is'];
+		$args['status_is_not']      = (array) $request['status_is_not'];
+		$args['product_includes']   = (array) $request['product_includes'];
+		$args['product_excludes']   = (array) $request['product_excludes'];
+		$args['variation_includes'] = (array) $request['variation_includes'];
+		$args['variation_excludes'] = (array) $request['variation_excludes'];
+		$args['coupon_includes']    = (array) $request['coupon_includes'];
+		$args['coupon_excludes']    = (array) $request['coupon_excludes'];
+		$args['tax_rate_includes']  = (array) $request['tax_rate_includes'];
+		$args['tax_rate_excludes']  = (array) $request['tax_rate_excludes'];
+		$args['customer']           = $request['customer'];
+		$args['refunds']            = $request['refunds'];
+		$args['attribute_is']       = (array) $request['attribute_is'];
+		$args['attribute_is_not']   = (array) $request['attribute_is_not'];
+		$args['category_includes']  = (array) $request['categories'];
```

Comment:
If we're making the change from `categories` to `category_includes`, does it make sense to also do that on the client too? I understand this may be a far reaching change and I'm not fully understanding why the change was made, but wondering if uniformity may be helpful in the future.

Reply:
This was definitely confusing to me when I started working on this new report.

These "param mappings" handle differences between what the REST endpoint Controller expects vs. the Query/DataStore that backs it. Keeping the mapping rather than doing a full rename everywhere allows any bookmarks users have created to function as expected. _(A use case I seem to remember us talking about a lot before launch)_

This particular change regarding `categories` needed to happen because the category filter on the Variations Report introduce excluding by category, which doesn't exist for other reports. The mapping takes care of the fact that the Categories Report hits the Products endpoint when specifying categories using `categories` and the Variations Report will either specify `category_includes` or `category_excludes` based on the filter value.